src/main/java/org/opensrp/web/controller/ActionController.java

```diff
@@ -53,24 +51,24 @@ public ActionController(ActionService actionService, ClientsRepository c, Alerts
     @ResponseBody
     public List<Action> getNewActionForANM(@RequestParam("anmIdentifier") String anmIdentifier, @RequestParam("timeStamp") Long timeStamp){
         List<org.opensrp.scheduler.Action> actions = actionService.getNewAlertsForANM(anmIdentifier, timeStamp);
-        return with(actions).convert(new Converter<org.opensrp.scheduler.Action, Action>() {
-            @Override
-            public Action convert(org.opensrp.scheduler.Action action) {
-                return ActionConvertor.from(action);
-            }
-        });
+        List<Action> actionList = new ArrayList<>();
```

Comment:
can we avoid this list ... may be something like https://stackoverflow.com/a/40035280

Reply:
> can we avoid this list ... may be something like https://stackoverflow.com/a/40035280

Code refactoring done, removed creating new list code